models/account.py

```diff
@@ -91,7 +91,7 @@ def update_account(
         email: str,
         azure_ad_subject_id: str,
         full_name: str,
-        roles: List[str],
+        roles: list[str],
```

Comment:
looks good. Any reason why are changing from typing `List` to just `list`

Reply:
Just newer type annotations, my IDE did this for me